src/commands/utility/sell-card.js

```diff
@@ -0,0 +1,54 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { UserClient } = require('../../database/clients/user-client');
+const { CardClient } = require('../../database/clients/card-client');
+const { RARITY_GROUP } = require('../../card-pack/rarity');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+	.setName('sellcard')
+	.setDescription('Sell a card for a credit.')
+	.addStringOption(option =>
+		option.setName('cardname')
+			.setDescription('The card to be sold for a credit. (Must be exact spelling)')
+			.setRequired(true)),
+	async execute(interaction) {
+		const userId = interaction.user.id;
+		const cardName = interaction.options.getString('cardname');
+		const userClient = new UserClient();
+		const cardClient = new CardClient();
+		const card = await cardClient.getCardByName(cardName);
+		try {
+			let removeCard = await userClient.removeCardFromUser(userId, card.card_id);
```

Comment:
@ccraven9

Reply:
So I think since removeCardFromUser() returns a promise or a boolean value false, a card cannot be lost.

There could be some better error handling written, so that if removeCardFromUser is unsuccessful, or encounters an error an exception is thrown that would be caught by the try-catch here in sell-card.js.